Source/WebCore/PAL/Configurations/PAL.xcconfig

```diff
@@ -71,7 +71,7 @@ SWIFT_OBJC_INTERFACE_HEADER_NAME = PALSwift-Generated.h;
 SWIFT_VERSION = 5.0;
 SWIFT_OBJC_INTEROP_MODE = objcxx;
 
-OTHER_SWIFT_FLAGS = $(OTHER_SWIFT_FLAGS_$(WK_PLATFORM_NAME));
+OTHER_SWIFT_FLAGS = $(OTHER_SWIFT_FLAGS_$(WK_PLATFORM_NAME)) -Xcc -DUCHAR_TYPE=char16_t;
```

Comment:
Might be preferable to use the native build setting:
```
GCC_PREPROCESSOR_DEFINITIONS = $(inherited) UCHAR_TYPE=char16_t;
```

and also to add it to CommonBase.xcconfig, so we don't have to worry about keeping this in sync as we add Swift-C++ code in more places.

Reply:
Yes - [here's the doom](https://ews-build.webkit.org/#/builders/131/builds/25376) from C files seeing that definition. I'll add a comment explaining this.